js/blocks/PitchBlocks.js

```diff
@@ -2233,6 +2233,7 @@ function setupPitchBlocks() {
             let delta = 0;
 
             function addPitch(note, octave, cents, frequency, direction) {
+                let transposition;
```

Comment:
I also came across this error. I am not sure if it's as simple as just declaring a variable. It will remove the error, but not sure about the desired behavior. It's the same as eliminating transposition from the equation altogether because it won't attain any value in this scenario.

Maybe something like:
`let transposition = 2 * delta;`
`if (turtle in logo.transposition) {`
                `transposition += logo.transposition[turtle];`
            `}`
needs to be done.
Again, not sure about this, otherwise would've addressed it.

Reply:
@aviral243  What do you mean by desired behavior? I have tested both 'note value  392 hertz' and 'milliseconds' by changing hertz and durations. It seems to work fine.